platform/stm32f4xx/export/platform/spi_bus.hpp

```diff
@@ -219,10 +220,38 @@ class spi_i2s_bus
     //! \tparam Iface_cfg Used for correct enable_if mechanism. Must not be passed by user.
     //! \return Status of operation.
     //!
-    template < ecl::i2s::data_format format, class Iface_cfg = spi_i2s_config >
+    template< ecl::i2s::data_format format, class Iface_cfg = spi_i2s_config >
     std::enable_if_t< Iface_cfg::m_bus_type == spi_bus_type::i2s, ecl::err >
     i2s_set_data_format();
 
+    //!
+    //! \brief Enables circular mode for the bus.
+    //! \details In circular mode when xfer is finished, the new xfer is started
+    //!  automatically without interruption. The same buffer is used.
+    //!  The the process continues until circular mode is disabled.
+    //!  This method must not be called during xfer().
+    //!
+    void enable_circular_mode();
+
+    //!
+    //! \brief Disables circular mode for the bus.
```

Comment:
Let's drop '\brief' since we are using [autobrief  feature](https://www.stack.nl/~dimitri/doxygen/manual/config.html#cfg_javadoc_autobrief).